docs/docs/reference/built-in-components/gatsby-slice.md

```diff
@@ -0,0 +1,166 @@
+
+By using the `<Slice>` React component in combination with the [`createSlice`](/docs/reference/config-files/actions/#createSlice) API for common UI features, Gatsby will be able to build and deploy individual pieces of your site that had content changes, not entire pages.
+
+## Faster builds in Gatsby Cloud
+
+With the introduction of [Incremental Builds](https://www.gatsbyjs.com/blog/2020-04-22-announcing-incremental-builds/), Gatsby Cloud has been able to reduce build times signficantly by only building the pages that changed. The `<Slice>` component helps reduce those builds times further.
+
+Common components that are shared across the majority of pages on your site might include a navigation bar, footer, or contact form. In today's frameworks, when you re-order the navigation bar items, the entire site needs to be rebuilt. However, if the navigation bar was created as a Gatsby Slice, the new navigation items only need to be built once and all pages will pull the new navigation bar when it's needed.
+
+## `createSlice` action
+
+The [`createSlice`](/docs/reference/config-files/actions/#createSlice) action from the [`createPages`](/docs/reference/config-files/gatsby-node/#createPages) API is the first step in creating a new Slice.
+
+```js:title=gatsby-node.js
+exports.createPages = async ({ actions }) => {
+  actions.createSlice({
+    id: `navigation-bar`,
+    component: require.resolve(`./src/components/navigation-bar.js`),
+    context: {
+      jokeOfTheDay: `What's blue and not heavy? Light blue.`,
+    }
+  })
+}
+```
+
+| Argument               | Type     | Description                                                                |
+| ---------------------- | -------- | -------------------------------------------------------------------------- |
+| `id` (Required)        | `string` | A unique identifier for this specific Slice. See also: [Aliases](#aliases) |
+| `component` (Required) | `string` | The path to the component being used as a Slice.                           |
+| `context`              | `object` | An object passed to the `component` as `sliceContext`.                     |
+
+## `Slice` component
+
+The Slice component requires an `alias` prop. Any props additional props will be passed to underlying component.
+
+| Prop               | Type     | Description                                                                                 |
+| ------------------ | -------- | ------------------------------------------------------------------------------------------- |
+| `alias` (Required) | `string` | The Slice created in `gatsby-node` to replace this component. See also: [Aliases](#aliases) |
+
+### Restrictions on using `Slice`
+
+#### Nested Slices
+
+Gatsby does not support nested Slices. This means if you have a high level `<Layout>` component as a slice, other Slices cannot exist within that `<Layout>` component anywhere in the tree.
+
+#### Props
```

Comment:
Shouldn't this be an h3?

Reply:
This is talking about restrictions of props (under the "Restrictions on using `Slice`" title).

Reply:
ahh 👍